packages/cli/jest.config.js

```diff
@@ -1,6 +1,6 @@
-//TODO: Somehow move this to the parent folder and have children use this config
+// TODO: Somehow move this to the parent folder and have children use this config
 /** @type {import('@ts-jest/dist/types').InitialOptionsTsJest} */
 module.exports = {
-  preset: 'ts-jest',
-  testEnvironment: 'node',
-};
\ No newline at end of file
+  preset: "ts-jest",
```

Comment:
Actually... Any reason for choosing double quotes instead of single quotes? Single quotes don't need to press shift

But quite mafan for you to change back also, idm just sticking to double quotes

Reply:
Double quotes are default, we can set everything to single quotes under .prettierrc then fix everything one shot